src/client/core.ts

```diff
@@ -28,7 +28,7 @@ export async function request<Req, Res>(options: ClientRequest<Req>, client: Cli
   const { url, method, body, contentType, params, overrides } = options;
   const headers: Record<string, string | AnyNumber | boolean | undefined> = {
     ...overrides?.HEADERS,
-    "x-aptos-client": `aptos-ts-sdk/${VERSION}`,
+    "x-aptos-client": `aptos-new-ts-sdk/${VERSION}`,
```

Comment:
How about putting `@aptos-labs/ts-sdk`?  That'll keep it named the same as the npm package, and forgo needing the `new` which will someday be old.

Reply:
Oh, yeah that's a good point... maybe something simpler that differentiates, but won't become too old:
`aptos-typescript-sdk`